resources/views/master.blade.php

```diff
@@ -0,0 +1,218 @@
+
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
@khanhpvrk What is the difference between `main.blade.php` and `master.blade.php`?